core/src/components/input/input.tsx

```diff
@@ -515,7 +515,71 @@ export class Input implements ComponentInterface {
   }
 
   private renderInput() {
-    return <Host>Stubbed input</Host>;
+    const mode = getIonMode(this);
+    const value = this.getValue();
+
```

Comment:
In other render implementations we access the property a single time across the function. Should we do the same thing here with `disabled`, `readonly`, and `inputId`?

e.g.:
```ts
const { disabled, readonly, inputId } = this;
```